### ISSUE TITLE
Staging Review finding: Radio group label is not visible nor explicit/descriptive

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128354]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments Team
**Product:** Appointments
**Feature:** Oracle Health Scheduling - Direct Schedule and Request for an Appointment

## Findings details
**VA.gov Experience Standard - issue:** [User encounters a component or pattern that isn't considered a best practice.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @kkitagawa-bosch (content)

### Description

On the "Which VA facility would you like to go to?" page, the provider radio group does not provide a visible label and the label repeats the content above. This may be confusing for users, as the content is repetitive and is a statement versus a label - e.g. "Providers", "Select a provider", etc.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="1056" height="314" alt="Unclear label" src="https://github.com/user-attachments/assets/176aef78-8cf0-4036-b747-0393486ee051" />

</details>

## Recommended action

Provide a visible label and ensure that it provides context to the radio group selection.

Keegan (content): I recommend updating the hint text to read "These facilities you're registered at offer ____. Select the facility you'd like to visit." 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 3.3.2 Labels or Instructions - Level A](https://www.w3.org/TR/WCAG22/#labels-or-instructions)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by Jenny-yw** [comment]
@ala-yna This is ready for dev ([figma file](https://www.figma.com/design/HnExAM5SGNXzUlv9yRS87t/Working-File-%7C-Appointments-FE?node-id=9665-8573)) cc @outerpress

**Comment by outerpress** [comment]
@Jenny-yw could you validate this one? Note one change to the radio list that came up when looking at https://github.com/department-of-veterans-affairs/va.gov-team/issues/128342 (switched to the "Tile" variant) that was intentional

**Comment by JunTaoLuo** [comment]
Hmm it's hard to tell whether this was deployed during the last daily deployment since it was so close to the code freeze deadline. I'll keep this open until the the daily deploys are restored this week.

**Comment by JunTaoLuo** [comment]
Closing as per [slack discussion](https://dsva.slack.com/archives/CSCD0B1V3/p1776106936430139). We'll validate next sprint that this is deployed to prod.

**Comment by JunTaoLuo** [comment]
Deployed to prod 4/15/26
